src/server/services/wms/qgswmsrenderer.h

```diff
@@ -297,6 +297,7 @@ namespace QgsWms
       bool featureInfoFromRasterLayer( QgsRasterLayer *layer,
                                        const QgsMapSettings &mapSettings,
                                        const QgsPointXY *infoPoint,
+                                       QgsRenderContext &renderContext,
```

Comment:
Could we keep this const?

Reply:
Thanks, done in https://github.com/qgis/QGIS/pull/55680/commits/e7c43d004d3fd7fd3dbe487c921d665e56665b0c .